### PR TITLE
Update Quantities notebook

### LINKED ISSUE #447: Quantities notebook in quarantine
As of PR #446, the quantities notebook appears out-of-date enough to [break Circle CI](https://circleci.com/gh/astropy/astropy-tutorials/724?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

We need a PR to bring it up to date or move it to the quarantine folder.

### PR DESCRIPTION
**Pull request by kakirastern**
Fixes #447. 

Update tutorial content to pass CI checks.

### CODE REVIEW

**Comment by kakirastern** [comment]
Looks like the `quantities` notebook is working now but the CI tests are plagued by apparent server-side errors.

**Comment by kakirastern** [comment]
Will need to try again during Learn Astropy cowork tomorrow...

**Comment by adrn** [comment]
Thanks @kakirastern - could you clear notebook outputs?

**Comment by kakirastern** [comment]
@adrn You are welcome. Sure, let me do it now...

File: tutorials/notebooks/quantities/quantities.ipynb [hunk 1]
```diff
@@ -368,7 +368,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "np.log10(M / u.Msun)"
+    "np.log10(M.to(u.Msun).value)"
```
Comment (adrn) [inline tutorials/notebooks/quantities/quantities.ipynb:371]: It's slightly better to use: `M.to_value(u.Msun)`

**Pull request merged**